apps/explorer/components/Contract/ContractHeader.tsx

```diff
@@ -1,32 +1,36 @@
 import {
   Badge,
-  ContractTimeline,
+  // ContractTimeline,
   LinkButton,
 } from '@siafoundation/design-system'
 import { ArrowLeft16, ArrowRight16 } from '@siafoundation/react-icons'
-import { SiaCentralContract } from '@siafoundation/sia-central-types'
-import { lowerCase } from '@technically/lodash'
 import { routes } from '../../config/routes'
 import { EntityHeading } from '../EntityHeading'
-import { siaCentral } from '../../config/siaCentral'
-import { to } from '@siafoundation/request'
+import { ExplorerFileContract } from '@siafoundation/explored-types'
 
 type Props = {
-  contract: SiaCentralContract
+  currentHeight: number
+  contract: ExplorerFileContract
   renewedToId?: string
   renewedFromId?: string
 }
 
+// This is dummy logic and I'm not even sure it makes sense, given the values.
+function contractStatus(valid: boolean, resolved: boolean) {
```

Comment:
Can be simplified to:


```suggestion
function contractStatus(valid: boolean, resolved: boolean) {
  return !valid ? 'invalid' : resolved ? 'resolved' : 'unresolved';
}
```